### PR TITLE
Misc changes (Part ??)

### PR DESCRIPTION
**Pull request by revam**
feel free to not merge this if you disagree with my changes. 😔

### CODE REVIEW

File: src/pages/collection/series/TmdbLinking.tsx
Comment (harshithmohan) [inline src/pages/collection/series/TmdbLinking.tsx:182]: Why? Why do you think it existed in the first place and why did you remove it?

File: src/pages/collection/series/TmdbLinking.tsx
Comment (harshithmohan) [inline src/pages/collection/series/TmdbLinking.tsx:225]: you could refactor the above if-else and this to have a single `createAutoLinks` line

File: src/components/Collection/Tmdb/AniDBEpisode.tsx
Comment (harshithmohan) [inline src/components/Collection/Tmdb/AniDBEpisode.tsx:37]: Is that more readable then "Unknown Airdate"?

**Review by harshithmohan — CHANGES_REQUESTED** [review]

File: src/components/Collection/Tmdb/AniDBEpisode.tsx
Comment (ElementalCrisis) [reply src/components/Collection/Tmdb/AniDBEpisode.tsx:37]: This will confuse people.

File: src/pages/collection/series/TmdbLinking.tsx
Comment (revam) [reply src/pages/collection/series/TmdbLinking.tsx:182]: It is no longer needed. In fact, it's getting in the way now. Because the server now distinguished between auto-matched (AKA "none") and overridden (AKA "user verified") non-links, where the override will prevent the automagic linking from changing the link, while the auto-matched will not. This change is just to brink the UI in line with what the server will return once the match is finalized.

File: src/pages/collection/series/TmdbLinking.tsx
Comment (revam) [reply src/pages/collection/series/TmdbLinking.tsx:225]: I'll move this change (after fixing it) and the above to their own PR.

File: src/components/Collection/Tmdb/AniDBEpisode.tsx
Comment (revam) [reply src/components/Collection/Tmdb/AniDBEpisode.tsx:37]: > Is that more readable than "Unknown Airdate"?

Please read the sauce code again — it says **Airdate Unknown**, not **Unknown air-date**. Impo, I found it a more coherent experience to display all "?"s in the same width and size as the formatted date, rather than using an improperly formed sentence with one made-up word ("airdate" instead of "air-date") and one proper word, written in reverse order, putting the descriptive term after the word it describes.

It was triggering my OCD _so_ **hard** that I submitted this PR. But then again, feel free to not merge this if you disagree with the change.

Comment (harshithmohan) [reply src/components/Collection/Tmdb/AniDBEpisode.tsx:37]: Made up?

https://www.merriam-webster.com/dictionary/airdate

Comment (harshithmohan) [reply src/components/Collection/Tmdb/AniDBEpisode.tsx:37]: You can change it to fix the order, but not '?'

Comment (hidden4003) [reply src/components/Collection/Tmdb/AniDBEpisode.tsx:37]: Bunch of question marks will lead to people report that something is broken, you can't show information like that to users.

Comment (revam) [reply src/components/Collection/Tmdb/AniDBEpisode.tsx:37]: > Made up?
> 
> https://www.merriam-webster.com/dictionary/airdate

Both my phone and pc doesn't recognise it as a word from it's included dictionary. I'll leave it at that.

Comment (revam) [reply src/components/Collection/Tmdb/AniDBEpisode.tsx:37]: > You can change it to fix the order, but not '?'

any suggestions?

Comment (harshithmohan) [reply src/components/Collection/Tmdb/AniDBEpisode.tsx:37]: Unknown airdate

Comment (harshithmohan) [reply src/components/Collection/Tmdb/AniDBEpisode.tsx:37]: > > Made up?
> > https://www.merriam-webster.com/dictionary/airdate
> 
> Both my phone and pc doesn't recognise it as a word from it's included dictionary. I'll leave it at that.

That's like saying you'll change "color" to "colour" because it's not recognized. I did check it in Google before putting "Airdate" there because I had the same doubt.

**Comment by ElementalCrisis** [comment]
So I take it we can close this PR since the changes are not happening?

**Pull request closed without merging**
